ansible/roles/data-products-deploy/templates/common.conf.j2

```diff
@@ -15,10 +15,10 @@ storage.secret.config="{{ dp_storage_secret_config }}"
 reports.storage.key.config="{{ dp_reports_storage_key_config }}"
```

Comment:
These are also storage variable names - `storage.key.config`, `storage.secret.config`, `reports.storage.key.config` & `reports.storage.secret.config`. Can we check this also?